src/main/java/org/wso2/carbon/module/swiftiso20022/validation/rules/AlphaNumericParamValidationRule.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.validation.rules;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONObject;
+import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationResult;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationRule;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidatorContext;
+
+import java.util.List;
+
+/**
+ * Alpha Numeric Param Validation Rule.
+ */
+public class AlphaNumericParamValidationRule extends ValidationRule {
+
+    private final List<ValidatorContext> validationParamList;
+    private static final String RULE_NAME = "Alpha Numeric Param Validation";
+    public AlphaNumericParamValidationRule(List<ValidatorContext> validationParamList) {
```

Comment:
add a new line before the constructor

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/02976704c3798e54f6e5b27d539a212b8bdb900d